e2e/wdio/core/tests/side-navigation.e2e-spec.ts

```diff
@@ -0,0 +1,92 @@
+import {
+    click,
+    getAttributeByName,
+    getElementArrayLength,
+    isElementDisplayed,
+    getElementClass,
+    refreshPage,
+    waitForPresent,
+    acceptAlert,
+    doesItExist
+} from '../../driver/wdio';
+import { sideNavigation } from '../../core/pages/side-navigation.po';
+
+describe('Side-navigation test suite', () => {
+
+    const snPage = new sideNavigation();
```

Comment:
variable names need to be meaningful and understandable. sideNavPage or sideNavigationPage